### DISCUSSION TITLE
Remote start missing?

### DISCUSSION THREAD

**Discussion by beckerben** [widewing/ha-toyota-na#95] [Q&A] [answered]
I can't figure it out, i have the subscription and see the remote start sensor, but i don't see the control for anything other than the "Lock" to toggle the door locks. I deleted and re-added the service as well, no luck, thoughts?

**Comment by Kelinni** [comment, marked as answer]
Hey Ben,

It’s a service you need to call. They are called actions now.  Create a script and type in perform an action. And then start typing Toyota.

[image0.png]
Sent from my iPhone

On Nov 21, 2024, at 2:08 PM, Ben Becker ***@***.***> wrote:

﻿

I can't figure it out, i have the subscription and see the remote start sensor, but i don't see the control for anything other than the "Lock" to toggle the door locks. I deleted and re-added the service as well, no luck, thoughts?

**Reply by beckerben** [reply]
Ah, figured i was missing something obvious, thank you!
